Lib/tarfile.py

```diff
@@ -1691,10 +1691,13 @@ class TarFile(object):
 
     extraction_filter = None    # The default filter for extraction.
 
+    record_size = RECORDSIZE    # The default record size, matches tar -b20
+
     def __init__(self, name=None, mode="r", fileobj=None, format=None,
             tarinfo=None, dereference=None, ignore_zeros=None, encoding=None,
             errors="surrogateescape", pax_headers=None, debug=None,
-            errorlevel=None, copybufsize=None, stream=False):
+            errorlevel=None, copybufsize=None, stream=False,
+            blocking_factor=None):
```

Comment:
Instead of `None`, should the blocking_factor default to 20, or a constant `BLOCKING_FACTOR`?